catroid/src/main/java/org/catrobat/catroid/CatroidApplication.java

```diff
@@ -49,6 +58,162 @@ public void onCreate() {
 		Log.d(TAG, "CatroidApplication onCreate");
 		settings = new ApplicationSettings(this);
 		CatroidApplication.context = getApplicationContext();
+		defaultSystemLanguage = Locale.getDefault().getLanguage();
+		// open the App in the last chosen language
+		languageSharedPreferences = getSharedPreferences("For_language", getApplicationContext().MODE_PRIVATE);
+		String langTag = languageSharedPreferences.getString("Nur", "");
```

Comment:
```java
String languageTag = languageSharedPreferences.getString("applicationLanguage", "");
```
Looking good now :+1:
Just one small improvement, create a final static string variable in Multilingual.java for `"applicationLanguage"` and always refer to this variable instead of using a hard-coded string.